wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -227,20 +238,20 @@ def _get_shebang_token(self) -> Optional[tokenize.TokenInfo]:
         return None
 
     def _check_executable_mismatch(self, shebang_token) -> None:
-        is_executable = os.access(self.filename, os.X_OK)
+        is_executable = file_is_executable(self.filename)
 
         if is_executable and shebang_token is None:
             self.add_violation(
-                ExecutableMismatchViolation(
-                    node=self.file_tokens[0],
+                ShebangViolation(
+                    node=self._violation_token,
```

Comment:
And after we inherit from `SimpleViolation` we would not be able to pass `node=` anymore.